packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -104,6 +111,30 @@ async function handler(req: NextApiRequest & { userId?: number }) {
     throw new HttpError({ statusCode: 400, message: "User not found" });
   }
 
+  // If it's just an attendee of a booking then just remove them from that booking
+  if (seatReferenceUId && bookingToDelete.attendees.length > 1) {
```

Comment:
Do not run through the normal canceling of a booking if there is more than 1 attendee. If there is only 1 attendee / the host is canceling the booking then it will be updated as `CANCELED`

Reply:
Will abstract this to a custom handle